### PR TITLE
Add information about API review schedule

### PR DESCRIPTION
**Pull request by terrajobst**
/cc @dotnet/fxdc @ericstj

### CODE REVIEW

**Review by bartonjs — APPROVED** [review]

File: docs/project/api-review-process.md [hunk 1]
```diff
@@ -38,6 +38,16 @@ APIs and some code samples that show how it should be used. If changes are neces
     replace the label `api-ready-for-review` with `api-needs-work`.
     * **Rejected**. In case we believe the proposal isn't a direction we want to go after, we simply write a comment and close the issue.
 
+## Review schedule
+
+ There are three methods to get an API review:
+
+* **Get into the backlog**. Generally speaking, filing an issue in `dotnet/runtime` and slapping the label `api-ready-for-review` on it will make your issue show up during API reviews. The downside is that we generally walk the backlog oldest-newest, so your issue might not be looked at for a while.
+* **Fast track**. If you need to bypass the backlog apply both `api-ready-for-review` and `blocking`. All blocking issues are looked at before we walk the backlog.
```
Comment (danmosemsft) [inline docs/project/api-review-process.md:46]: did you mention that `blocking` was confused (eg with `blocked`) ? Would it reduce, or increase confusion if we renamed it `blocking-other-work` or similar?

@karelz

File: docs/project/api-review-process.md [hunk 2]
```diff
@@ -38,6 +38,16 @@ APIs and some code samples that show how it should be used. If changes are neces
...
+* **Dedicated review**. If your issue needs an hour or longer, send an email to FXDC and we book dedicated time. Rule of thumb: if your API proposal has more than a dozen APIs or your APIs have complex policy, then you need 60 min or more. When in doubt, send mail to FXDC.
```
Comment (danmosemsft) [inline docs/project/api-review-process.md:47]:
```suggestion
* **Dedicated review**. If your issue needs an hour or longer, and you're in the .NET team, send an email to FXDC and we book dedicated time. Rule of thumb: if your API proposal has more than a dozen APIs or your APIs have complex policy, then you need 60 min or more. When in doubt, send mail to FXDC.
```
I don't like writing "only if you're in the .NET team" but that's your intent here.

**Review by danmoseley — APPROVED** [review]
thanks for adding this

File: docs/project/api-review-process.md [hunk 3]
```diff
@@ -38,6 +38,16 @@ APIs and some code samples that show how it should be used. If changes are neces
     replace the label `api-ready-for-review` with `api-needs-work`.
     * **Rejected**. In case we believe the proposal isn't a direction we want to go after, we simply write a comment and close the issue.
 
+## Review schedule
+
+ There are three methods to get an API review:
+
+* **Get into the backlog**. Generally speaking, filing an issue in `dotnet/runtime` and slapping the label `api-ready-for-review` on it will make your issue show up during API reviews. The downside is that we generally walk the backlog oldest-newest, so your issue might not be looked at for a while.
```
Comment (ericstj) [inline docs/project/api-review-process.md:45]:
```suggestion
* **Get into the backlog**. Generally speaking, filing an issue in `dotnet/runtime` and applying the label `api-ready-for-review` on it will make your issue show up during API reviews. The downside is that we generally walk the backlog oldest-newest, so your issue might not be looked at for a while.
```

File: docs/project/api-review-process.md (hunk 2 above)
Comment (ericstj) [reply docs/project/api-review-process.md:47]: If not on the .NET team and you think this applies, find a sponsor based on area-owners.md.

Comment (terrajobst) [reply docs/project/api-review-process.md]: I reword it to:

> * **Dedicated review**. This only applies to area owners. If an issue you are the area owner for needs an hour or longer, send an email to FXDC and we book dedicated time. Rule of thumb: if the API proposal has more than a dozen APIs and/or the APIs have complex policy, then you need 60 min or more. When in doubt, send mail to FXDC.
